Campus-iOS/GradesComponent/Model/AverageGrade.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AverageGrade.swift
+//  Campus-iOS
+//
+//  Created by Anton Wyrowski on 08.05.23.
+//
+
+import Foundation
+
+struct AverageGradeStudien: Decodable {
+    let studium: [AverageGrade]
+}
+
+struct AverageGrade: Decodable, Identifiable {
+    public var id: String {
+        "\(studyId)-\(averageGradeWeightedByCredits)"
+    }
+    public var studyId: String
+    public var studyDesignation: String
+    public var averageGradeWeightedByCredits: String
```

Comment:
What actually happens for "Erstis" that don't have any grades yet? What value will be received from the API here? How do we handle the visualization in the app?

Reply:
If the API does not return an average grade, we hide the whole average grade section. Otherwise, we display the API's response.